internal/core/algorithm/faiss/Capi.cpp

```diff
@@ -31,9 +31,11 @@ FaissStruct* faiss_create_index(
     const int m,
     const int nbits_per_idx,
     const int metric_type) {
-  //printf(__FUNCTION__);
-  //printf("\n");
-  //fflush(stdout);
+#if DEBUG
+  printf(__FUNCTION__);
+  printf("\n");
+  fflush(stdout);
+#endif
```

Comment:
```suggestion
#if DEBUG
  printf("%s\n", __FUNCTION__);
  fflush(stdout);
#endif
```
You can use some macros such as `__PRETTY_FUNCTION__` and `__LINE__` for detail debug message.

FYI, there is one of some debugging techniques like below.
```
#ifdef DEBUG
#define DEBUG_MESSAGE(x) do { std::cout << "[" << __FUNCTION__ << ":" << __LINE__ << "]" << x << std::endl; } while(0)
#else
#define DEBUG_MESSAGE(x)
#endif
```